src/snac/data/Contributor.php

```diff
@@ -193,25 +193,26 @@ public function fromArray($data) {
      *
      * @param \snac\data\Contributor $other Other object
      * @param boolean $strict optional Whether or not to check id, version, and operation
+     * @param boolean $checkSubcomponents optional Whether or not to check SNACControlMetadata, nameEntries contributors & components
```

Comment:
Just thought of this: you can add yourself as an '@author' to the top of these files :+1: